### ISSUE TITLE
How to close a case via API

### ISSUE THREAD

**Issue by mthlvt** [TheHive-Project/TheHive4py#67]
It may be obvious, but how do I close an existing case via thehive4py (v 1.4.2) please? I tried to update a case with the following fields without success:
```python
api = TheHiveApi(....)

hiveCase = api.case(caseId)

hiveCase.status='Resolved'
hiveCase.resolutionStatus='TruePositive'
hiveCase.impactStatus='NoImpact'
hiveCase.summary='closed by api'
hiveCase.tags=['test']

hiveResponse = api.update_case(hiveCase)

if hiveResponse.status_code == 200:
    logging.warning(json.dumps(hiveResponse.json(), indent=4, sort_keys=True))
else:
    logging.warning('ko: {}/{}'.format(hiveResponse.status_code, hiveResponse.text))
```

Thanks in advance

**Comment by nadouani** [comment]
Fixed by #57

you can specify the fields you can to update like below:

```
api.update_case(case, ['status', 'resolutionStatus', 'impactStatus', 'tags', 'summary'])
```

**Comment by mthlvt** [comment]
Tried, updated fields but didn't update the status 
```python
hiveResponse = api.update_case(hiveCase,['Resolved','TruePositive', 'NoImpact',['xxx'],'xxx'])

if hiveResponse.status_code == 200:
    logging.warning(json.dumps(hiveResponse.json(), indent=4, sort_keys=True))
```

result:
```json
    "owner": "xxxx", 
    "resolutionStatus": "TruePositive", 
    "severity": 2, 
    "startDate": 1517816109000, 
    "status": "Open", 
    "summary": "xxxx", 
    "tags": [
        "xxx", 
        "yyy", 
        "test"
    ], 
    "title": "xxxx`", 
    "tlp": 2, 
    "updatedAt": 1518011095568, 
    "updatedBy": "xxxx"
}

```
